CHANGES.md

```diff
@@ -1,3 +1,29 @@
+## 1.5.1 2018-10-24
+
+Linkerd 1.5.1 adds an exciting new interpreter to its suite of interpreter module. The new 
+`io.l5d.consul.interpreter` now allows Linkerd to read dtabs directly from a Consul KV store agent
+that This effectively removes the dependence on Namerd to resolve service names to concrete names
```

Comment:
I'd reword this away from saying that it removes the dependence on Namerd.  Instead, I'd say that using this interpreter allows Linkerd to read dtabs from Consul KV directly instead of using Namerd.